src/views/Fridge.vue

```diff
@@ -98,13 +109,25 @@ export default class Fridge extends Vue {
     this.products = this.products.filter(p => p.name != product.name);
     await Firestore.instance.removeFromStorage(product);
     await Firestore.instance.addToShoppingList(product);
+
+    this.alertMessage = `${product.name} was added to the Shopping List`;
+    setTimeout(() => {
+      this.alertMessage = "";
+    }, 3000);
```

Comment:
Would be nice to have this in a separate method. Something like `showAlert(message, timeout)`